ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/EvaluatorImpl.java

```diff
@@ -83,6 +83,9 @@ public String evaluate(String source) throws BallerinaShellException {
     public ShellCompilation getCompilation(String source) {
         PackageCompilation compilation;
         ExceptionStatus exceptionStatus;
+
+        // need to do this first to address fails of tree parser and snippet factoryee
```

Comment:
Typo here in the comment

Reply:
Removed with [8b92cd3](https://github.com/ballerina-platform/ballerina-lang/pull/35871/commits/8b92cd3e5a1f36b03359d62bc0261290cfcd921f).